api/agent_test.go

```diff
@@ -1070,15 +1070,16 @@ func TestAPI_AgentForceLeave(t *testing.T) {
 	}
 }
 
-func TestAPI_AgentForceLeavePrune(t *testing.T) {
+func TestAPI_AgentForceLeaveWithOpts(t *testing.T) {
```

Comment:
Also maybe it's worth to add test for `forceLeaveWithOpts` with datacenter specified